tor/main.py

```diff
@@ -46,9 +47,12 @@ def run(config):
 
 
 def main():
-    build_bot('bot', __version__, full_name='u/ToR')
+    config.debug_mode = bool(os.environ.get('DEBUG_MODE', False))
+    bot_name = 'debug' if config.debug_mode else os.environ.get('BOT_NAME', 'bot')
```

Comment:
if all of the bots are running on one machine, do we want to change BOT_NAME to something more specific to u/tor?

Reply:
We can set the environment variable name on invocation and it only affects that instance, not the rest of the bots.